client/src/Components/Dialog.js

```diff
@@ -33,11 +24,10 @@ export default function NavDialog({ label }) {
 				placeSelf: 'center'
 			}}
 		>
-			<Button onClick={handleClickOpen}>{label}</Button>
 			<Dialog
 				fullScreen={fullScreen}
 				open={open}
-				onClose={handleClose}
+				onClose={event => close(false)}
```

Comment:
`close ` takes no arguments, so this should be  `onClose={close}`

Reply:
done